chaosazure/machine/actions.py

```diff
@@ -160,6 +224,15 @@ def __fetch_machine_at_random(filter, configuration, secrets):
     return choice
 
 
+def __get_os_type(machine):
+    os_type = machine['properties']['storageProfile']['osDisk']['osType']\
+        .lower()
+    if os_type == OS_WINDOWS or os_type == OS_LINUX:
+        return os_type
+    else:
+        raise FailedActivity("Unknown OS Type: %s" % os_type)
+
```

Comment:
A slightly cleaner way to write these lines:

```
if os_type not in (OS_LINUX, OS_WINDOWS):
     raise FailedActivity("Unknown OS Type: %s" % os_type)

return os_type
```

Reply:
will change it, thanks! such remarks always welcome